### PR TITLE
Introduction To Git lesson: Fix link text

### PR DESCRIPTION
**Pull request by JohnGCT**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

The link in "Additional Resources" for "Git and GitHub in plain English" was broken. I found a working link to the same article and changed it.

**2. This PR:**

 -Deleted https://blog.red-badger.com/blog/2016/11/29/gitgithub-in-plain-english
 -Added https://content.red-badger.com/resources/what-is-git-and-github

**3. Additional Information:**

### CODE REVIEW

File: git/foundations_git/introduction_to_git.md [hunk 1]
```diff
@@ -54,6 +54,6 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   [Git and GitHub in plain English](https://blog.red-badger.com/blog/2016/11/29/gitgithub-in-plain-english)
+*   [Git and GitHub in plain English](https://content.red-badger.com/resources/what-is-git-and-github)
```
Comment (01zulfi) [inline git/foundations_git/introduction_to_git.md:57]: I thnk we should change the link text to reflect the article heading
```suggestion
*   [What is Git and GitHub?](https://content.red-badger.com/resources/what-is-git-and-github)
```

**Review by 01zulfi — CHANGES_REQUESTED** [review]

File: git/foundations_git/introduction_to_git.md (hunk 1 above)
Comment (JohnGCT) [reply git/foundations_git/introduction_to_git.md:57]: Indeed ! Thanks for pointing it out.
